sentry/src/main/java/io/sentry/SentryExceptionFactory.java

```diff
@@ -62,11 +62,13 @@ public SentryExceptionFactory(final @NotNull SentryStackTraceFactory sentryStack
    * @param exceptionMechanism The optional {@link Mechanism} of the {@code throwable}. Or null if
    *     none exist.
    * @param thread The optional {@link Thread} which the exception originated. Or null if not known.
+   * @param snapshot if the captured {@link java.lang.Thread}'s stacktrace is a snapshot.
```

Comment:
`l`: Maybe a short explanation of what a snapshot is would be lovely. A link to `SentryStackTrace.snapshot` would do the job.

Reply:
SentryExceptionFactory is an internal class, private API.
`SentryStackTrace.snapshot` already has a user friendly javadocs, a link makes sense